velox/functions/prestosql/ArrayIntersectExcept.cpp

```diff
@@ -269,29 +271,143 @@ class ArrayIntersectExceptFunction : public exec::VectorFunction {
 
   // If there's a `constantSet`, whether it refers to left or right-hand side.
   const bool isLeftConstant_{false};
-};
+}; // class ArrayIntersectExcept
+
+template <typename T>
+class ArraysOverlapFunction : public exec::VectorFunction {
+ public:
+  ArraysOverlapFunction() {}
+
+  explicit ArraysOverlapFunction(
```

Comment:
explicit is only needed for a single-argument constructor; this constructor has 2 arguments, hence, explicit is not needed.